### ISSUE TITLE
0.61.x Discussion

### ISSUE THREAD

**Issue by grabbou** [react-native-community/releases#140]
Conversation on this thread are limited to 0.61.x issues and cherry-pick requests from commits that are already on master.

An example of a good such request is a bug fix for a serious issue that has been merged into master but did not make the 0.61.x cut.

In other words, if you cannot point to a particular commit on master, then your request likely belongs as a new issue in http://github.com/facebook/react-native/issues.

If the commit you request to cherry-pick is a complicated port, you will be asked to create the PR to the 0.61-stable branch yourself, in order to ensure that the process proceeds smoothly

**Comment by TheSavior** [comment]
I know we aren't thinking about cherry picks yet, but it would be great if we are able to get this into 0.61 https://github.com/facebook/react-native/commit/b8d6ef372663fe6d467144abfc5d2c9352dc28d6

**Comment by kelset** [comment]
should we just wait until master is green and re-cut after that?

**Comment by grabbou** [comment]
I haven't been able to get any updates regarding CI failures for the past few days and it's not clear when they will be fixed. 

After discussing it on the #releases channel, we have decided to ship it "as is" because it works locally and it's an RC.  

It's a requirement to cherry-pick CI fixes before we go stable, but we want to ship some of the new features out the door.

**Comment by grabbou** [comment]
I have managed to dig deeper and fix a few tests. I am now working on fixing `use_framework!` tests to make sure we ship a release that works 100%.

I will get back with the update, once fixed, we are good to go.

**Comment by fkgozali** [comment]
The following should be picked:
* https://github.com/facebook/react-native/commit/947e71a922c0db5d3d3780d249d1a8d183534c22
* https://github.com/facebook/react-native/commit/59f84ca1fd4a5700d67cc7c68194dd845a1ecfd6

It should fix some OOM issue when having a lot of images (especially GIFs)

**Comment by grabbou** [comment]
I have picked them @fkgozali. The release is now being published. I have managed to resolve CircleCI issues by contacting their support team.

**Comment by grabbou** [comment]
That said, I have also unlocked this thread now that the release is going to be available really soon.

**Comment by gaodeng** [comment]
Since iOS 13 & Android 10 will be officially released soon, I really hope that this commit https://github.com/facebook/react-native/pull/26172 about Dark Mode can be merged quickly.

**Comment by owinter86** [comment]
Not sure if this is a bug, or the [pro tip regarding the ios splash screen](https://facebook.github.io/react-native/docs/running-on-device#pro-tips) is no longer valid. 

But if it is implemented, the splash screen is permanently rendered above the app and is never dismissed.

**Comment by kelset** [comment]
@owinter86 	can you please open an issue for it?

**Comment by owinter86** [comment]
> @owinter86 can you please open an issue for it?

No worries, I can do that, but this is directly related to the 0.61RC though and the (releases)[https://github.com/facebook/react-native/releases] page directs me to this issue for bug reports?

**Comment by kelset** [comment]
yes, but ideally we'd prefer to have here only the link to the issue; that way the conversation/investigation/debugging of that issue can happen over there and not here.

**Comment by jamsch** [comment]
Just got it working on a large app. Two things I noticed

1. Gestures using `react-native-gesture-handler@1.4.1` (react-navigation-drawer) doesn't seem to work. Assuming there's some native breaking changes?. No build or console errors. Haven't taken the time to properly look in to this.

2. Image borders (borderWidth/borderColor) doesn't seem to work on the RC. 

```js
function AvatarImage({ uri }: AvatarImageProps) {
    return <Image source={{ uri }} style={styles.avatar} />;
}

const styles = StyleSheet.create({
 avatar: {
    borderColor: Colors.white,
    borderRadius: 50,
    borderWidth: 3,
    height: 90,
    width: 90,
    overflow: "hidden",
  },
}
```

0.60.5

![Selection_060](https://user-images.githubusercontent.com/12927717/63903857-1c7c6c00-ca63-11e9-838a-46605cf4f98e.png)

0.61.0-rc.0

![Selection_061](https://user-images.githubusercontent.com/12927717/63903869-243c1080-ca63-11e9-957f-b18cf9d6f1d7.png)

**Comment by kelset** [comment]
@BrendonSled can you open an issue for it & post the link here?

**Comment by BrendonSled** [comment]
@kelset https://github.com/facebook/react-native/issues/26258

**Comment by IljaDaderko** [comment]
I believe I found an issue when Archiving iOS build https://github.com/facebook/react-native/issues/26259

TL;DR There is a cli step that is run and apparently `ios` is invalid target, perhaps a type there?

**Comment by gaearon** [comment]
Redbox is broken: https://github.com/facebook/react-native/issues/26260. This is a release blocker. Don't have a fix yet.

**Comment by gaearon** [comment]
Once D17139310 and D17140543 land, we need to cherry pick them.

**Comment by sinodko** [comment]
This would be super nice to cherry pick facebook/react-native#51aacd5

**Comment by gaearon** [comment]
@grabbou

Please pick:

* https://github.com/facebook/react-native/commit/f107d3b78c95ed0313b39e55976cf6b6be36775e 
* https://github.com/facebook/react-native/commit/42a385bddf895be5b00dd30f82396f0d8197cfc1

Thank you.

**Comment by owinter86** [comment]
Not sure if this is directly related to @TheSavior comment about the TouchableNativeFeedback issue.

> [This commit](https://github.com/facebook/react-native/commit/2f7732b1458109605a1a29c58d2a7ca38aed552e) which I _believe_ is included in 0.61 made a breaking change to TouchableNativeFeedback and the revert will need to get picked once it lands. I'm leaving on vacation today before RN.EU and might not be able to flag the actual commit.

But this release has caused some unusual behaviour when using  `TouchableNativeFeedback.SelectableBackgroundBorderless()` 

https://github.com/facebook/react-native/issues/26289

**Comment by TheSavior** [comment]
Here is a link to the TouchableNativeFeedback revert commit: https://github.com/facebook/react-native/commit/c526b3ff14c245ad6c853632c945e61c02eecffe

@owinter86, I’m not sure if that is related. I’d be surprised if it was, but I’m not sure anything else has touched the component than my commit, so maybe it is. :-) I’ll investigate when I try to reland my change.

**Comment by jamsch** [comment]
Looks like https://github.com/facebook/react-native/issues/26241 was introduced in the RC. Haven't tried iOS yet.

**Comment by estevaolucas** [comment]
Please pick https://github.com/facebook/react-native/commit/82a8080f0704e83079d0429e4e367f5131052e64 and https://github.com/facebook/react-native/commit/a58dd9683da6135bca119a8b0ae5d0ea5382cf4c. 

Those commits will allow [Flipper's Layout plugin](https://fbflipper.com/docs/features/layout-plugin.html) to be used on iOS projects. 

(more about it here: https://github.com/facebook/react-native/pull/26053 cc @janicduplessis @axe-fb)

**Comment by grabbou** [comment]
React Native 0.61.0-rc.2 is now on CircleCI and should hit npm soon.

@sinodko, unfortunately, it sounds like your cherry-pick request is a feature request, rather than a bug fix. We have decided to not cherry-pick it yet.

**Comment by tlupo** [comment]
I’m not sure if it’s still possible but it would be great to include facebook/react-native@88e18b6 (by @janicduplessis) to fix blob memory leak on Android (https://github.com/react-native-community/releases/issues/130#issuecomment-511917694). Thank you very much !

**Comment by fkgozali** [comment]
https://github.com/facebook/react-native/commit/241091d527ae7e1866c3f9bdf067cde146e377fa fixes iOS test with use_frameworks, would be good to pick to keep 0.61 RC branch green.

**Comment by 0x7fs** [comment]
Please consider facebook/react-native@c9708fb9c4c8262b93b0f32755b1d5966808de4b and facebook/react-native@7f3a0e831c5953cacbd65d731d62747e07d8cf85 for the next RC. These fix building with `use_frameworks!` without requiring third party libraries to add `s.static_framework = true` to their Podspecs.

See PR with more information here: https://github.com/facebook/react-native/pull/26151

**Comment by gabrielfurini** [comment]
Hey guys, I was struggling with the same issue as on https://github.com/facebook/react-native/pull/26360.
Machines with case sensitive disks are not able to find the `podspec` file for Yoga. Please consider merging it too.

**Comment by estevaolucas** [comment]
What about those other commits to make 0.61.0 and its template to support Flipper (including on RNTester):
* https://github.com/facebook/react-native/commit/0165489b8fc29423dfde958d23103e28cb523240
* https://github.com/facebook/react-native/commit/0a5c43fcc37640e1568f2d04563d232db2f532d8
* https://github.com/facebook/react-native/commit/70274f4e9166dbe9da2d04953a95c2909b455d1d
* https://github.com/facebook/react-native/commit/755ad3b33a40bfb901d8b943c46c429433945713
* https://github.com/facebook/react-native/commit/adac04601ec36d35eca42532ec824c5790dc71ed

**Comment by grabbou** [comment]
Going to cherry-pick commits and release a new version today. Thanks @gabrielfurini reporting that - I guess my Mac is case-insensitive then, since we were able to run it w/o problems while testing.

**Comment by MoOx** [comment]
@grabbou this very tiny (but important) fix in the flowconfig of the template would be nice to have https://github.com/facebook/react-native/commit/e6b2cf04188fc9647bae4bef4cca5d4dde22a657

**Comment by hassanbazzi** [comment]
> Going to cherry-pick commits and release a new version today. Thanks @gabrielfurini reporting that - I guess my Mac is case-insensitive then, since we were able to run it w/o problems while testing.

I was gonna walk up to you in the callstack conf and office and tell you about this. haha. Let me know if you need help testing at all. I assume you had a loooooooooong week at the conf.

**Comment by grabbou** [comment]
@0x7fs I have cherry-picked these commits that fall under this PR. @fkgozali, this is exactly what you have discussed within the PR comments. Hopefully, `static_framework`'s work without issues now.

@tlupo, please consider checking the commit you are pointing to whether it's already part of the release you are commenting on. Not really a deal-breaker, but Github UI shows this information right below the description of the commit and would save me a few minutes while doing this cherry-picking process. I hope that information helps while testing!

@estevaolucas, I am double-checking with Facebook whether releasing this under 0.61 is okay. Few people mentioned it being part of 0.62, so want to make sure we don't conflict in this process. Will report back and if that's not the case, will release another RC the day after.

**Comment by grabbou** [comment]
Update: all commits are already on the CI and I am testing it right now. The release should be out there in a bit. I will be back with the updates after I finish another round of the meetings.

**Comment by a-c-sreedhar-reddy** [comment]
But ubuntu users cant user Flipper right?

**Comment by fkgozali** [comment]
> I am double-checking with Facebook whether releasing this under 0.61 is okay. Few people mentioned it being part of 0.62, so want to make sure we don't conflict in this process. Will report back and if that's not the case, will release another RC the day after.

We chatted about this briefly. I think we should let 0.62 have the support naturally instead of picking to 0.61. This is aligned with what @grabbou mentioned up top:

> An example of a good such request is a bug fix for a serious issue that has been merged into master but did not make the 0.61.x cut.

Picking Flipper support (more of a new feature) could delay the 0.61 RC stabilization and potentially more work for everyone here to fix the potential issues forward in the RC itself.

**Comment by axemclion** [comment]
@a-c-sreedhar-reddy - yes, Flipper does not current seem to have a build for Linux, but Flipper is an electron app, and is Open source. Would you be interested in adding linux builds ?

**Comment by Jarred-Sumner** [comment]
After upgrading from `0.60.5` to `0.61.0-rc.3`, I am running into this error:

```
[fatal][tid:com.facebook.react.ExceptionsManagerQueue] Unhandled JS Exception: TypeError: hmrClient.send is not a function. (In 'hmrClient.send(JSON.stringify({
        type: 'log-opt-in'
      }))', 'hmrClient.send' is undefined)
```

![image](https://user-images.githubusercontent.com/709451/64653380-25fac200-d3db-11e9-8ca9-923ad0263557.png)

I have updated my Podfile to rename these two pods: (they were `React-RCTWebSocket` and `React-DevSupport` before I think)

```ruby
pod 'React-Core/DevSupport', :path => '../node_modules/react-native/'
pod 'React-Core/RCTWebSocket', :path => '../node_modules/react-native/'
```

Is anyone else seeing this?

Adding a `debugger` [here](https://github.com/facebook/react-native/blob/master/Libraries/Utilities/HMRClient.js#L164) shows that `hmrClient` is defined but `hmrClient.send` is not a function (as the error says)

now I'm looking through where `send` should be defined and I will update this issue when I see it

**Comment by a-c-sreedhar-reddy** [comment]
@axemclion I created PR updating docs regarding building for linux from source.
Where can I add linux builds?

**Comment by gedeagas** [comment]
Hi guys, how about this commit ? https://github.com/facebook/react-native/commit/3915c0fa6178fbb8aa9b8bd54881cfdbde8fde70 this commit fix custom fontWeight numeric values for Text on Android

**Comment by kelset** [comment]
@gedeagas, as @grabbou mentioned above, you can check if a commit is already in a release in its page.

The one you are linking, in particular, seem to be already in:

<img width="628" alt="Screenshot 2019-09-11 at 16 25 41" src="https://user-images.githubusercontent.com/16104054/64711050-ee5e4980-d4b0-11e9-8755-8ee7891313f6.png">

**Comment by jrwpatterson** [comment]
Any idea when this is going to go live... (I know all time scales are flexible) We just have something that for the life of me will not work in RN60 but works perfectly first time in RN61 and my managers want it stable (and I want to stop working on getting it to work on 60 that I'm fairly convinced at this stage isn't going to work!)

**Comment by marcorm** [comment]
Hi all,
i'm using RN 0.61.0-rc.3 on a side project.
I'm facing [this issue](https://github.com/kmagiera/react-native-reanimated/issues/381
) with react-native-reanimated.

It should be caused by [this commit](https://github.com/facebook/react-native/commit/1e45a41dc2d4699d4e439994e81ca5ab82d630b3).

Just wanted to inform someone here...

**Comment by tamirrab** [comment]
https://github.com/react-native-community/releases/issues/140#issuecomment-531509868

Same here

**Comment by MoOx** [comment]
@jrwpatterson haste module system is gone, so you need to reference full path the Alert module now.

**Comment by xiao99xiao** [comment]
Just found this issue https://github.com/facebook/react-native/issues/26473 introduced in RN 0.61.0-rc.3 , caused by this commit https://github.com/facebook/react-native/commit/bd2b7d6c0366b5f19de56b71cb706a0af4b0be43 .

**Comment by grabbou** [comment]
We will need to wait for:
- https://github.com/facebook/react-native/pull/26416 
- https://github.com/kmagiera/react-native-reanimated/issues/381
- https://github.com/facebook/react-native/issues/26473 (investigate)

to release another release candidate.

Since we are aiming for releasing a new version later this week, we should finalize the release.

**Comment by a-c-sreedhar-reddy** [comment]
new RC or new version?

**Comment by grabbou** [comment]
New release candidate and then, promoted automatically to stable after a few days (no cherry-picking in the meantime)

**Comment by SpawnAtis** [comment]
RNGH doesn't work with rc.3 on Android 
- [https://github.com/kmagiera/react-native-gesture-handler/issues/745]( https://github.com/kmagiera/react-native-gesture-handler/issues/745)
 -  [https://github.com/react-native-community/react-native-tab-view/issues/879](https://github.com/react-native-community/react-native-tab-view/issues/879)

**Comment by acoates-ms** [comment]
This one is required for react-native-windows to be able to update to 61
[facebook/react-native#26462](https://github.com/facebook/react-native/pull/26462)

**Comment by kmagiera** [comment]
I'll be investigating issues related to reanimated and gesture-handler. Thanks for reaching out

**Comment by bartolkaruza** [comment]
I'll have a look at this one facebook/react-native#26473 (Modal dismiss issue)

**Comment by bartolkaruza** [comment]
Fix for the above issue is in this PR: https://github.com/facebook/react-native/pull/26490

If someone could have a look, we can cherry pick that commit in safely after merging of the PR.

**Comment by janicduplessis** [comment]
Reland this android blob fix which had to be reverted before this branch was cut.

https://github.com/facebook/react-native/commit/9b2374b542f87b7baefcfb4a3eb4f57029069b57

**Comment by kmagiera** [comment]
Tracked dow the gesture handler issue. The change that cause the problem is this one https://github.com/facebook/react-native/commit/d0792d4b8ac42711dfd9fccb782f16e72ce3e335 as it changes the way root view is instantiated. Previously we were overriding react native's default rootview in order to forward touch events into a gesture handler processing pipeline. I'll be working on finding the best way to fix this issue that hopefully does not involve patching RN nor changing user's current setup, but don't know yet if that's possible.

A temporary workaround for the problem is to use `gestureHandlerRootHOC` to wrap top level component of your RN app.

**Comment by janicduplessis** [comment]
@kmagiera Oh I remember that. I didn't have the time to investigate and just ended up reverting this commit in my fork. I think it is just a bug in the diff as it should not have changed any functionality, it just refactored some logic in a shared class if I remember correctly.

**Comment by kmagiera** [comment]
yeah, I see your comment from June on that commit now

**Comment by grabbou** [comment]
Thanks @kmagiera and @bartolkaruza for sending the patches for the issues! 

Following up on my previous comment [here](https://github.com/react-native-community/releases/issues/140#issuecomment-532175844):

- <s>facebook/react-native#26416</s> (picked)
- <s>facebook/react-native#26462</s> (picked)
- kmagiera/react-native-reanimated#381 (investigate)
- facebook/react-native#26473

There's work already happening in the last two, waiting for updates and I will let you know.

**Comment by gaodeng** [comment]
Any chance we can get this cherry picked,It fix iOS 13 status bar style issue https://github.com/facebook/react-native/commit/796b3a1f8823c87c9a066ea9c51244710dc0b9b5

**Comment by gaearon** [comment]
It would be good if we could pick https://github.com/facebook/react-native/commit/ee5cb5a880015c58a98f4cdcd3f3cc978196246d. It fixes a significant memory regression we've found in our apps.

**Comment by marcorm** [comment]
Guys, forgive me if it's not the right place where to post it.
I noticed a problem with [react-native-paper](https://github.com/callstack/react-native-paper) and react-native 0.61.x.

When i open a `Dialog` inside a `Portal` it shows up weird.
Same problem with FAB.

The problem doesn't arise with RN 0.60.5.
It happens with the last versions of react-native-paper (stable and alpha).

Here are two attachments showing the issue:
RN 0.60.5
<img src="https://user-images.githubusercontent.com/2567466/65372403-1d727900-dc70-11e9-9097-432a795dcacc.png" width="250" height="500">
RN 0.61.x
<img src="https://user-images.githubusercontent.com/2567466/65372442-a5f11980-dc70-11e9-895b-86643fe42050.png" width="250" height="500">

P.S. don't pay attention to Icons in the bottom tab.
P.P.S. i'll post it also in RNP rep

**Comment by dulmandakh** [comment]
- https://github.com/facebook/react-native/commit/4763000554c71be40b82a740651706781acad2e3
- https://github.com/facebook/react-native/commit/be2a2529af21414e645f2b04c5a1c819828ecda6

**Comment by mikebouwmans** [comment]
> Guys, forgive me if it's not the right place where to post it.
> I noticed a problem with [react-native-paper](https://github.com/callstack/react-native-paper) and **react-native 0.61.x** on **Android API Level 26**.
> Doesn't happen on API Level 23 or 29.
> 
> When i open a `Dialog` inside a `Portal` it shows up weird.
> Same problem with FAB.
> 
> The problem doesn't arise with RN 0.60.5.
> It happens with the last versions of react-native-paper (stable and alpha).
> 
> Here are two attachments showing the issue:
> RN 0.60.5
> <img alt="" width="250" height="500" src="https://user-images.githubusercontent.com/2567466/65372403-1d727900-dc70-11e9-9097-432a795dcacc.png">
> RN 0.61.x
> <img alt="" width="250" height="500" src="https://user-images.githubusercontent.com/2567466/65372442-a5f11980-dc70-11e9-895b-86643fe42050.png">
> 
> P.S. don't pay attention to Icons in the bottom tab.
> P.P.S. i'll post it also in RNP rep

This is not an react-native-paper issue, I have this aswell when using 0.61-rc.3, something with shadows is broken

**Comment by kelset** [comment]
@mikebouwmans can you open an issue in the main repo with the details about it (and then post the link here)? So that there is a "recorded" bug report over there

**Comment by spyshower** [comment]
May I ask when is the next rc/stable version coming?

**Comment by mikebouwmans** [comment]
> @mikebouwmans can you open an issue in the main repo with the details about it (and then post the link here)? So that there is a "recorded" bug report over there

Ok I will try to do that tonight

**Comment by marcorm** [comment]
> Ok I will try to do that tonight

@mikebouwmans 
Maybe this can help: https://github.com/callstack/react-native-paper/issues/1341#issuecomment-534155733

**Comment by mikebouwmans** [comment]
> > Ok I will try to do that tonight
> 
> @mikebouwmans
> Maybe this can help: [callstack/react-native-paper#1341 (comment)](https://github.com/callstack/react-native-paper/issues/1341#issuecomment-534155733)

Well, I had shadow/borderRadius issues even without react-native-paper. I'm trying to reproduce it in a new app but can't seem to get the bug.

**Comment by kelset** [comment]
> May I ask when is the next rc/stable version coming?

You can, but we may not have an answer ;)
Mike has been busy lately so maybe by end of week we can do another round of cherry picks & release another RC, but I can't give fixed dates.

**Comment by mikebouwmans** [comment]
> @mikebouwmans can you open an issue in the main repo with the details about it (and then post the link here)? So that there is a "recorded" bug report over there

I've created an issue on the main repo, see: facebook/react-native#26544

**Comment by jeffreyffs** [comment]
@kelset I believe that issue was fixed with this MR that was merged 5 days ago: https://github.com/facebook/react-native/pull/26495

So we probably want to cherrypick: https://github.com/facebook/react-native/commit/9f0dede1c913612e1241432f4cbccdc74d23a1e4

Though maybe @kmagiera can confirm that everything is working now :)

**Comment by grabbou** [comment]
As I said in my previous comment, we need to resolve all remaining bugs and issues before this release gets stable.

The https://github.com/kmagiera/react-native-reanimated/issues/381 still remains unresolved and I wanted to get it fixed before this gets stable. Reasons: the library is quite popular and we should always look out for the effects of the release on the community libraries.

However, given that this issue is still unresolved and it only affects the development mode, I am starting to think whether we should release it "as is" or wait.

I let @kmagiera as the owner of the library comment whether this can be fixed in the userland or another release will be needed from our side.

**Comment by grabbou** [comment]
@jeffreyffs the commit you are referring to fixes another (but equally important) problem and has been already picked.

**Comment by grabbou** [comment]
I just spoke to @kmagiera on a side and he said that fix for https://github.com/kmagiera/react-native-reanimated/issues/381 is on the way and is scheduled to land in a matter of day(s).

Since it will land in the userland and does not affect the release modes, I think we are good to safely ignore this error and proceed with 0.61.

**Comment by grabbou** [comment]
@gaodeng, we are working on keeping the cherry-picks to critical issues only. I identified your commit as a non-critical addition that can be safely released under 0.62, which we will soon begin to work on.

**Comment by grabbou** [comment]
Shipping, closing and opening a new thread.

**Comment by felippepuhle** [comment]
@grabbou will https://github.com/facebook/react-native/commit/0cafa0f5d1e7fa5369b765f4b97f38bf1608230a be included on next 0.61 patch release or just on 0.62? thanks!

**Comment by ASchwad** [comment]
According to the new release of react-native 0.61.1 they fixed something with react-native-gesture-handler. Is the HOC still required to use?

**Comment by bneigher** [comment]
I'm still having issues with this on 0.61.1 (no HOC) / (didn't try HOC)

**Comment by dulmandakh** [comment]
- Fix gradle script - https://github.com/facebook/react-native/commit/7c8e266e390de71ce39233b53a369b2c5e8a1c14

**Comment by piuholo** [comment]
As mentioned, version 0.61.1 does not fix the critical issue with `react-native-gesture-handler`.

**Comment by kmagiera** [comment]
@piuholo can you please describe in a little bit more details what the "critical issue" is? Which OS does it affect, which version of OS and RN. Preferably if you could make a simple repo from react-native init with gesture handler that we could run to see the issue that would be very helpful.

**Comment by piuholo** [comment]
@kmagiera In my case, it was the issue described here

[https://github.com/kmagiera/react-native-gesture-handler/issues/320](https://github.com/kmagiera/react-native-gesture-handler/issues/320)

that had resurfaced on iOS after upgrading to RN `0.61.X` from `0.60.X`. However, I was able to work around the problem with the import statement found in the same issue thread.

**Comment by kelset** [comment]
Folks, sorry to bother you but there is a dedicated issue for 0.61.1 [here](https://github.com/react-native-community/releases/issues/146).

Also, please always don't report issues directly on these issues but just comment linking to an issue from the main repo - it helps not losing track of them
